cmd/harmony/config_migrations.go

```diff
@@ -156,4 +156,14 @@ func init() {
 		confTree.Set("Version", "2.0.0")
 		return confTree
 	}
+
+	migrations["2.0.0"] = func(confTree *toml.Tree) *toml.Tree {
+		// Legacy conf missing fields
+		if confTree.Get("Pprof.Folder") == nil {
```

Comment:
Can we also set enabled default to true? Can we also add default value to ProfileNames, ProfileIntervals, and ProfileDebugValues.